src/relay/op/make_op.h

```diff
@@ -62,6 +62,9 @@ Expr MakeRepeat(Expr data, int repeats, int axis);
 
 Expr MakeReshape(Expr data, Array<Integer> newshape);
 
+Expr MakeReshapeLike(Expr lhs, Expr rhs, int64_t lhs_begin, Integer lhs_end, int64_t rhs_begin,
```

Comment:
Why {lhs,rhs}_begin is int64_t and {lhs,rhs}_end is Integer?

Reply:
I did this because the beginning index (in both cases) must always be an integer, but the end index can be `None` which means I must use a nullable `Integer` wrapper. I could make everything `Integer` and check that beginning is always defined. I did feel a bit weird using `int64_t` directly since everything else seem to use `int` but the value wrapped by `Integer` is `int64_t` so that's why I chose it.

Reply:
I noticed that the Attrs I defined uses `int` and not `int64_t` so I'll probably just use `int` since other code mostly uses it.